apify-api/openapi/paths/request-queues/request-queues@{queueId}@requests@unlock.yaml

```diff
@@ -0,0 +1,55 @@
+post:
+  tags:
+    - Storage/Request queues/Requests locks
+  summary: Unlock requests
+  description: |
+    Unlocks requests in the queue for the client. The requests are unlocked if were locked by the same client key or if locked from the same Actor run.
```

Comment:
I'm not sure I understand what this actually does. Does it unlock all requests locked from the same run, *and* (in addition to) all requests locked by the same client key (I assume to account for working with the RQ from a script outside of the platform)?

Reply:
If the API is called from run it unlocks all requests locked by this run. If it is called outside of the run it unlocks requests locked by the same clientKey.

Reply:
I updated the description.